lib/msal-browser/src/config/Configuration.ts

```diff
@@ -2,15 +2,20 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { AuthOptions, SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
+import { SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
 import { BrowserUtils } from "../utils/BrowserUtils";
 import { BrowserConstants } from "../utils/BrowserConstants";
 
 // Default timeout for popup windows and iframes in milliseconds
 const DEFAULT_POPUP_TIMEOUT_MS = 60000;
 const DEFAULT_IFRAME_TIMEOUT_MS = 6000;
 
-export type BrowserAuthOptions = AuthOptions & {
+export type BrowserAuthOptions = {
```

Comment:
@pkanher617 @sameerag FYI had to make this change to because common clients take `authority: Authority`.

This also now has the benefit that `NodeAuthOptions` doesn't expore `redirectUri`, `postLogoutRedirectUri`, and `NavigateToLoginRequestUrl`, which are not applicable, and thus makes the public API cleaner.

Reply:
I am fine with this.